pkg/option/option.go

```diff
@@ -132,6 +174,26 @@ func (opt *Options) YAML() string {
 	return opt.yamlStr
 }
 
+// UseInitialCluster returns true if the cluster.initial-cluster is defined. If it is, the ClusterJoinUrls is ignored.
+func (opt *Options) UseInitialCluster() bool {
+	return len(opt.Cluster.InitialCluster) > 0
```

Comment:
is it goroutine safe? a mutex is used in `SetInitialCluster` to update `opt.cluster.InitialCluster`.